web/packages/teleport/src/stores/storeNotifications.ts

```diff
@@ -0,0 +1,61 @@
+/**
+ * Copyright 2023 Gravitational, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Store } from 'shared/libs/stores';
+
+type NoticeKinds = 'access-lists';
+
+export type Notice = {
+  kind: NoticeKinds;
+  id: string;
+  resourceName: string;
+  date: Date;
+  route: string;
+};
```

Comment:
I'm wondering if we could make it more generic. Currently, we are requiring from all notification kinds to have things like `resourceName` and `route`. 
I'm pretty sure that in the future we will have items that are not related a resource or doesn't link to anything.

Maybe we could do something more in this shape:
```ts
type AccessListNotification = {
  kind: 'access-lists';
  resourceName: string;
  route: string;
};

export type Notification = {
  item: AccessListNotification;
  id: string;
  date: Date;
};
```